runhouse/resources/hardware/sagemaker_cluster.py

```diff
@@ -930,7 +930,7 @@ def _refresh_ssm_session_with_cluster(self, num_ports_to_try: int = 5):
             self._create_ssm_session_with_cluster()
 
         # https://github.com/aws-samples/sagemaker-ssh-helper/blob/main/sagemaker_ssh_helper/sm-connect-ssh-proxy
-        full_module_name = "scripts/sagemaker_cluster/refresh-ssm-session.sh"
+        full_module_name = "resources/hardware/scripts/refresh-ssm-session.sh"
```

Comment:
all these scripts are sagemaker specific, should we create another subfolder within `hardware`? will matter more if we add other cluster scripts at some point

Reply:
yea that makes sense, scripts is a little generic here. just clarifying, are you thinking:

1. `resources/hardware/sagemaker` that contains the scripts + sagemaker_cluster.py
2. `resources/hardware/scripts/sagemaker` with sagemaker scripts, and the sagemaker_cluster.py stays in `hardware/`
3. something else

this might also be subject to change if we do add other cluster/resource scripts but should be easy to move around

Reply:
good point, for now feels a little overkill to bundle the scripts with the cluster file, so I think option 1 makes sense